Directory.Packages.props

```diff
@@ -81,11 +81,11 @@
     <PackageVersion Include="NuGet.Client.EndToEnd.TestData" Version="1.0.0" />
     <PackageVersion Include="NuGet.Core" Version="2.14.0-rtm-832" />
     <PackageVersion Include="NuGetValidator" version="2.1.1" />
-    <PackageVersion Include="Portable.BouncyCastle" Version="1.9.0" />
     <PackageVersion Include="SharpZipLib" Version="1.4.2" />
     <PackageVersion Include="System.Collections" Version="$(SystemPackagesVersion)" />
     <PackageVersion Include="System.CommandLine" Version="$(SystemCommandLineVersion)" />
     <PackageVersion Include="System.ComponentModel.Composition" Version="$(SystemComponentModelCompositionPackageVersion)" />
+    <PackageVersion Include="System.Formats.Asn1" Version="7.0.0" />
```

Comment:
This version is marked as having a known vulnerability: https://www.nuget.org/packages/System.Formats.Asn1/9.0.0-preview.6.24327.7#versions-body-tab

Reply:
Good catch!